tests/integration/fixtures.py

```diff
@@ -403,6 +413,51 @@ def _cleanup_changesets(changesets: List[str]) -> None:
 # Helpers for Cfn
 
 
+@pytest.fixture
+def deploy_cfn_template(
+    cfn_client,
+    lambda_client,
+    cleanup_stacks,
+    cleanup_changesets,
+    is_change_set_created_and_available,
+    is_stack_created,
+):
+    stack_name = f"stack-{short_uid()}"
+    change_set_name = f"change-set-{short_uid()}"
+    state = {}
+
+    def _deploy(template_body_or_file, **kwargs):
+        candidates = [template_body_or_file, template_path(template_body_or_file)]
+        for template_file in candidates:
+            if os.path.exists(template_file):
+                template_body_or_file = load_file(template_file)
+                break
+        template_rendered = render_template(template_body_or_file, **kwargs)
+
+        response = cfn_client.create_change_set(
+            StackName=stack_name,
+            ChangeSetName=change_set_name,
+            TemplateBody=template_rendered,
+            ChangeSetType="CREATE",
+        )
+        change_set_id = response["Id"]
+        stack_id = response["StackId"]
+
+        wait_until(is_change_set_created_and_available(change_set_id))
+        cfn_client.execute_change_set(ChangeSetName=change_set_id)
+        wait_until(is_stack_created(stack_id))
+
+        state.update({"stack_id": stack_id, "change_set_id": change_set_id})
```

Comment:
might make sense to allow for this fixture to deploy multiple stacks in which case we'd need to change the state var to a list